src/azure-cli/azure/cli/command_modules/mysql/_util.py

```diff
@@ -374,19 +375,18 @@ def _resolve_api_version(client, provider_namespace, resource_type, parent_path)
 
 
 def run_subprocess(command, stdout_show=None):
+    command = shlex.split(command)
```

Comment:
The best practice is to construct `command` as a `list` from the very beginning.

Reply:
Does the POSIX mode need to be mentioned therefore developers may notice the potential difference when splitting their cmds?

Agreed on constructing the list of cmd at the beginning

Reply:
The `command` string is constructed by `mysql` module itself:

https://github.com/Azure/azure-cli/blob/9119731bf5940981c3fe1f65ae495699cdb4b019/src/azure-cli/azure/cli/command_modules/mysql/_util.py#L416

The developer of `mysql` should be aware they should follow POSIX convention to build the `command` string.